### PR TITLE
Fix e2e tests for regional

### PR DESCRIPTION
**Pull request by wojtek-t**

### CODE REVIEW

File: test/e2e/framework/test_context.go [hunk 1]
```diff
@@ -245,6 +246,7 @@ func RegisterClusterFlags() {
 	flag.StringVar(&cloudConfig.Zone, "gce-zone", "", "GCE zone being used, if applicable")
 	flag.StringVar(&cloudConfig.Region, "gce-region", "", "GCE region being used, if applicable")
 	flag.BoolVar(&cloudConfig.MultiZone, "gce-multizone", false, "If true, start GCE cloud provider with multizone support.")
+	flag.BoolVar(&cloudConfig.MultiMaster "gce-multimaster", false, "If true, multi-master cluster is being used.")
```
Comment (wojtek-t) [inline test/e2e/framework/test_context.go]: Maybe this should be gke-multimaster? WDYT?
Any better suggestion?

Comment (shyamjvs) [reply test/e2e/framework/test_context.go]: Could you mention GKE in the flag description to indicate it's used by it?

Comment (wojtek-t) [reply test/e2e/framework/test_context.go]: Done

**Comment by wojtek-t** [comment]
PTAL

**Pull request merged**